dev/integration_tests/wide_gamut_test/ios/Runner/DummyPlatformView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  DummyPlatformView.swift
+//  Runner
+//
+//  Created by Jonah Williams on 11/21/23.
+//
+
+import Foundation
+
+import Flutter
+import UIKit
+
+class FLNativeViewFactory: NSObject, FlutterPlatformViewFactory {
+    private var messenger: FlutterBinaryMessenger
+
+    init(messenger: FlutterBinaryMessenger) {
+        self.messenger = messenger
+        super.init()
+    }
+
+    func create(
+        withFrame frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: Any?
+    ) -> FlutterPlatformView {
+        return FLNativeView(
+            frame: frame,
+            viewIdentifier: viewId,
+            arguments: args,
+            binaryMessenger: messenger)
+    }
+
+    public func createArgsCodec() -> FlutterMessageCodec & NSObjectProtocol {
+          return FlutterStandardMessageCodec.sharedInstance()
+    }
+}
+
+class SolidColorView: UIView {
+  private var _started: Bool = false;
```

Comment:
Above you don't use the underscore prefix.  I don't think its required in the swift style.

Reply:
Done